doc/source/reference/ufuncs.rst

```diff
@@ -328,9 +328,10 @@ advanced usage and will not typically be used.
     and an error in a future release.
 
     If 'out' is None (the default), a uninitialized return array is created.
-    The output array is then filled with the results of the ufunc in the
-    places that 'where' is True, by default everywhere if 'where' is not
-    specified. Note that outputs not explicitly filled are left with their
+    The output array is then filled with the results of the ufunc in the places
+    that the broadcast 'where' is True. If 'where' is the scalar True (the
+    default), then this corresponds to the entire output being filled.
```

Comment:
Both here and above, I'd omit the "(...)" and use commas, e.g., ", the default, ".

Reply:
ok

Reply:
I am new to github, so if I am missing something please correct me, but I believe the documentation for what the function returns should also be qualified with additional information.

The current documentation states that the function returns the negative of the input `x`:

> Returns :
>> y : ndarray or scalar
>> Returned array or scalar: y = -x.

Referring to my own usage as requested, I would not have thought that I should read documentation that had no immediate relevance to what I was trying to accomplish, namely the documentation for `out`.

I would have seen what the function returns and I would have looked where to apply that conditionally on the input, so I would have read and understood the `returns` and `where` portion.

I would not have presumed that the *lack* of `out` (being an optional parameter) would fundamentally alter what the function returns, just as I would not presume (even now) that `casting`, `order`, `dtype`, etc. or the lack thereof would cause the function to become both non-deterministic and return values uncorrelated to the input.

Does that make sense? I might have done something like this as a first-time user:
```
>>> import numpy as np
>>> x = np.array([-1, 1, -1, 1])
>>> np.negative(x) # checking the basic functionality
array([ 1, -1,  1, -1])
>>> t = np.array([True, False, True, False])
>>> np.negative(x, where=t)
array([              1, 139927355638704,               1, 139927355550720])
```
At this point I would be at a loss to explain how such a large number is returned in place of a `-1`, and since I have not used the `out` parameter at any point, I would not have expected that to have had any bearing on the behavior and would not expect adding an `out` parameter to correct the behavior.

Reply:
On digging deeper, it seems the [recoomendation](https://numpydoc.readthedocs.io/en/latest/format.html#sections) (scroll down to **4. Parameters**), it seems parenthesis is the way to go

Reply:
>  I would not have thought that I should read documentation that had no immediate relevance to what I was trying to accomplish

That documentation _is_ linked to under `kwargs` at least.